docs/courses/sct.md

```diff
@@ -0,0 +1,59 @@
+## Submission Correctness Tests
+
+Automated and meaningful feedback is an important aspect of the learning experience on DataCamp. When studens make a mistake in a coding exercise, we want to tell them what they are doing wrong. The life and blood of the automated system that points out mistakes and guides students to the correct solution is the Submission Correctness Test, or SCT.
+
+The SCT is a script of custom tests specified for every coding exercise. These custom tests have access to the code students submitted and the output and workspace they created with their code. For every taught language, there is an open-source library that provides a wide range of functions to verify these elements of a student submission. When the functions spot a mistake, they will automatically generate a meaningful feedback message.
```

Comment:
Not sure whether `every` is the right term? As not all exercises require them?

Reply:
every coding exercise does, right?